packages/app-store/applecalendar/pages/setup/index.tsx

```diff
@@ -4,9 +4,7 @@ import { useForm } from "react-hook-form";
 import { Toaster } from "react-hot-toast";
 
 import { useLocale } from "@calcom/lib/hooks/useLocale";
```

Comment:
The v2 pages above say they're deleted, but they're actually moved to the location of v1 (so without the v2 prefix), the diffs you see in those original v1 pages are actually the changes that were applied in v2.